admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -145,6 +146,7 @@ public ReplicationField updateReplication(
     setIfPresent(config::setDestination, destinationId);
     setIfPresent(config::setFilter, filter);
     setIfPresent(config::setBidirectional, biDirectional);
+    config.setModified();
```

Comment:
:question: Why not automatically set the modified timestamp inside the `save()` of all config objects?

Reply:
I assume that replication running on Ion would want to be able to save the config that it receives with the modified time that is already on the config. If I just timestamp it in the `save()` that timestamp will be overwritten.

Reply:
really good point. However, for Ion to be able to do that, the graphql endpoint would need to expose the ability to update the timestamp externally. Worst case, the timestamp would be updated automatically, Ion would reload the config, see it has changed and simply update Ion's DB; not a big issue

Reply:
Aren't we updating the DB through the ReplicationPersistentStore? if we set the modified date in the `save()` method there, then we will have to find some other way to save the configs with the modified date we have.

Reply:
I am saying it might not matter that if Ion sends an update to config and that timestamp is updated to the latest which is newer than the remote. Actually, I might prefer that as I won't have to worry about time shifts between nodes.

Reply:
If I'm reading this right, it seems odd that we would allow the modified time to be updated via the graphql endpoint even though no other fields in the configuration changed.

Reply:
That's a good point. I suppose somebody could make an update request with all null values, then we'd change the modified date for no good reason

Reply:
it goes beyond that, we would also be saving it back to the DB when in fact we shouldn't. I agree with @peterhuffer but the issue is bigger. We shouldn't be saving to the DB something that hasn;'t changed and if we don't then the modified timestamp wouldn't be updated which is why it shouldn't be under control of the client but under control of the save() operation that should be able to tell if anything has changed.